inprocess/src/test/java/io/grpc/inprocess/InProcessTransportTest.java

```diff
@@ -55,23 +61,26 @@ public class InProcessTransportTest extends AbstractTransportTest {
   private static final String TRANSPORT_NAME = "perfect-for-testing";
   private static final String AUTHORITY = "a-testing-authority";
   protected static final String USER_AGENT = "a-testing-user-agent";
+  private static final int TIMEOUT_MS = 5000;
+  private static final long TEST_MESSAGE_LENGTH = 100;
 
   @Rule
   public final GrpcCleanupRule grpcCleanupRule = new GrpcCleanupRule();
 
   @Override
   protected InternalServer newServer(
-      List<ServerStreamTracer.Factory> streamTracerFactories) {
-    InProcessServerBuilder builder = InProcessServerBuilder
-        .forName(TRANSPORT_NAME)
-        .maxInboundMetadataSize(GrpcUtil.DEFAULT_MAX_HEADER_LIST_SIZE);
-    return new InProcessServer(builder, streamTracerFactories);
+      int port, List<ServerStreamTracer.Factory> streamTracerFactories) {
+    return newServer(streamTracerFactories);
   }
 
   @Override
   protected InternalServer newServer(
-      int port, List<ServerStreamTracer.Factory> streamTracerFactories) {
-    return newServer(streamTracerFactories);
+      List<ServerStreamTracer.Factory> streamTracerFactories) {
+    InProcessServerBuilder builder = InProcessServerBuilder
+        .forName(TRANSPORT_NAME)
+        .maxInboundMetadataSize(GrpcUtil.DEFAULT_MAX_HEADER_LIST_SIZE)
+        .assumedMessageSize(TEST_MESSAGE_LENGTH);
```

Comment:
This means that we aren't testing without assumedMessageSize.  I would suggest that instead of setting assumedMessageSize here we just create a new server in basicStreamInProcess()
```
    InProcessServerBuilder builder = InProcessServerBuilder
        .forName(TRANSPORT_NAME)
        .maxInboundMetadataSize(GrpcUtil.DEFAULT_MAX_HEADER_LIST_SIZE)
        .assumedMessageSize(TEST_MESSAGE_LENGTH);
server = new InProcessServer(builder, server.getStreamTracerFactories());
```

Reply:
Sure.